doc/howto.md

```diff
@@ -134,8 +134,8 @@ function initSouthbound(callback) {
 }
 ```
 
-This Express code sets up a HTTP server, listening in the 8080 port, that will handle incoming requests targeting 
-path `/iot/d` using the middleware `manageULRequest()`. This middleware will contain all the Southbound logic, and 
+This Express code sets up a HTTP server, listening in the 8080 port, that will handle incoming requests targeting
+path `/iot/d` using the middleware `manageULRequest()`. This middleware will contain all the Southbound logic, and
```

Comment:
This middleware will contain all the Southbound logic?

Reply:
Fixed - 376c3e2 - Replaced with *logic south of ...*